1.4/Defs/Hediffs/Hediffs_FormerHuman.xml

```diff
@@ -158,6 +160,7 @@
 		<label>permanently feral</label>
 		<description>[PAWN_nameDef] used to be a human, but the mechanites in [PAWN_possessive] system have forgotten his prior form — [PAWN_pronoun] is now sealed in this form for the rest of [PAWN_possessive] life. Reversion serums will not work.</description>
 		<hediffClass>HediffWithComps</hediffClass>
+		<preventsPregnancy>true</preventsPregnancy>
```

Comment:
I realize this is a delicate topic, but I believe the intention in the past was that once someone goes permaferal they'd be allowed to mate with others of their new species, allowing a highly-unethical means of acquiring new breeding specimens.  There's even an exception in the patch specifically to allow permaferals to mate.

It's not possible for permaferals to "snuggle" with humans, so there shouldn't be a worry of them causing human/animal pregnancies unlike with normal former humans.

Reply:
From what I remember, i currently remove the hediff from perm feral to make them a normal animal in all ways.

This was because the sapience meter remained and could be refilled after.

It is something we should test, and also whether or not you can or should be able to tag permanently feral.

But i think this member is specifically related to biotech humanlike reproduction and has no influence on animals?

Reply:
> But i think this member is specifically related to biotech humanlike reproduction and has no influence on animals?

Ah, well *that's* confusing.  Thanks, Tynan -w-

Testing would be good.  And as far as tagging permaferals, I believe it was originally forbidden for balance reasons, but I personally think it makes sense to allow it for the same reason as allowing breeding -- it requires you to permanently sacrifice a pawn (or unlucky prisoner) to get a permaferal.  Not much different than Ripscanning in biotech.  Perhaps the error for non-perma-ferals could be that the mechanites are still too active and are interfering with the tagging process.

Reply:
Okay, i looked into it and i was wrong on all accounts.
The field also applies to animals and I don't remove the hediff but only the former human comp.